### PR TITLE
Cypress app actions with Angular

### PR DESCRIPTION
**Pull request by culas**
Describes how to use app actions with an Angular app.

### CODE REVIEW

File: drafts/cypress-app-actions-with-angular.md [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+# The Problem
+
+Due to the nature of the application in question, many tests required multiple steps to reach the desired start conditions.
+The existing tests would go through the same parts of the UI again and again.
+Other tests would rely on a pre-defined set of test data, which is left in an inconsistent state after an error.
+That caused the following tests that relied on the same data to fail too.
+All in all, this led to slow tests and was often the source of flakiness on the CI.
+
+Completely reorganising the end-to-end tests for an optimised flow through the application was not a viable option.
+We simply didn't have the time for it (or rather: it wasn't a high enough priority), and even then, we wouldn't be able to eliminate all issues.
```
Comment (r3dDoX) [inline drafts/cypress-app-actions-with-angular.md:27]: Not entirely sure, but I think this should be:
```suggestion
We simply didn't have the time for it (or rather: it wasn't a high enough priority), and even then, we wouldn't have been able to eliminate all issues.
```

File: drafts/cypress-app-actions-with-angular.md [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+    const urlPath = 'items';
+    cy.backendRequest('post', urlPath, itemName);
+}
+```
+
+# Conclusion
+
+Once we set this app action up, the affected tests greatly improved stability, readability, and speed.
+Using app actions halved the duration of some tests.
+Unexpected side effects and resolved most unwanted dependencies.
```
Comment (r3dDoX) [inline drafts/cypress-app-actions-with-angular.md:128]: There is something missing in this sentence. What happend to the unexpected side effects? Maybe even combine it with the sentence before it

**Review by r3dDoX — COMMENTED** [review]
Great post!

File: drafts/cypress-app-actions-with-angular.md (hunk 2 above)
Comment (culas) [reply drafts/cypress-app-actions-with-angular.md]: Removed "side effects" part, it's unclear and a bit redundant.

**Review by r3dDoX — APPROVED** [review]

**Pull request merged**